python/src/scipp/compat/mantid.py

```diff
@@ -366,7 +368,10 @@ def convert_Workspace2D_to_data_array(ws, **ignored):
     return array
 
 
-def convert_EventWorkspace_to_data_array(ws, load_pulse_times=True, **ignored):
+def convert_EventWorkspace_to_data_array(ws,
+                                         load_pulse_times=True,
+                                         realign_events=False,
```

Comment:
I don't believe the `realign_events` option is tested anywhere?

Reply:
Fixed. This did however highlight missing comparison ops of `Run` and `Sample`, I opened #1039 for this (needs a fix in Mantid).